### PR TITLE
chore: Update to Guidelines

### PR DESCRIPTION
**Pull request by jjfrench**

### CODE REVIEW

File: docs/source/technical_tutorials/visualization/stac_ipyleaflet.ipynb [hunk 1]
```diff
@@ -1,70 +1,101 @@
 {
```
Comment (smk0033) [inline docs/source/technical_tutorials/visualization/stac_ipyleaflet.ipynb:1]: <p>Only thing I would do (which I think we already discussed in Slack) is to change the title to use one #. Everything else looks good to me, though!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (jjfrench) [reply docs/source/technical_tutorials/visualization/stac_ipyleaflet.ipynb:1]: Oops, thought I had saved that change 😂

**Review by emmalu — APPROVED** [review]
👍🏽

**Comment by smk0033** [comment]
Changes also look good to me!

**Pull request merged**